Dockerfile.rhel7

```diff
@@ -7,5 +7,6 @@ FROM registry.ci.openshift.org/ocp/4.16:cli-artifacts as cli-artifacts
 
 FROM registry.ci.openshift.org/ocp/4.16:cli
 COPY --from=builder /go/src/github.com/openshift/must-gather/collection-scripts/* /usr/bin/
-COPY --from=cli-artifacts /usr/share/openshift/linux_amd64/oc.rhel8 /usr/bin/oc
+COPY --from=cli-artifacts /usr/bin/oc /usr/bin/oc
```

Comment:
@sferich888 we don't have to copy a specific oc arch here. The cli-artifacts container has the rhel8 variant of oc in /usr/bin/oc. Since the build system builds on native HW, we will always get the right binary arch.

https://github.com/openshift/oc/blob/master/images/cli-artifacts/Dockerfile.rhel#L15-L21

Reply:
We are trying to change the default oc's base image to RHEL9 and default oc in cli-artifacts will also not work on must-gather.
/hold

Reply:
Proposed this https://github.com/openshift/must-gather/pull/420 instead